services/identity/service.go

```diff
@@ -229,7 +223,7 @@ func (s *Service) ProposeVote(si *network.ServerIdentity, v *ProposeVote) (netwo
 		}
 		_, err = s.propagateSkipBlock(sid.Root.Roster, usb, propagateTimeout)
 		if err != nil {
-			return nil, err
+			return nil, sda.NewClientErrorCode(4200, err.Error())
```

Comment:
same for here, between 4200 and 4100, would be nice to have clear variable name if possible.